.github/workflows/security-check.yaml

```diff
@@ -0,0 +1,52 @@
+name: Security Checks
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+env:
+  FAIL_LEVEL: WARNING
+  ENV_TYPE: pipenv
+  DEP_PATH: src/
+  APP_PATH: src/
+  EXTRA_ARGS: "--settings=registrar.config.settings"
+  DJANGO_SECRET_KEY: not-a-secret-jw7kQcb35fcDRIKp7K4fqZBmVvb+Sy4nkAGf44DxHi6EJl
+  DATABASE_URL: "postgres://not_a_user:not_a_password@not_a_host"
+  DJANGO_BASE_URL: "https://not_a_host"
+
+jobs:
+  security-check:
+    name: Django security check
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out
+        uses: actions/checkout@v3
+      - name: Scan Django settings for security issues
+        id: check
+        uses: victoriadrake/django-security-check@master
+      - name: Upload output
+        uses: actions/upload-artifact@v2
+        with:
+          name: security-check-output
+          path: output.txt
+
+  owasp-scan:
+    name: OWASP security scan
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out
+        uses: actions/checkout@v3
+      - name: OWASP scan
+        run: docker compose run owasp
+        working-directory: ./src
+      - name: Upload output
+        uses: actions/upload-artifact@v2
+        with:
+          name: owasp-scan-output
+          path: ./src/zap_report.html
```

Comment:
This is not working as intended, perhaps because the path is wrong or maybe because the file is only being written within a container.

Reply:
Removed, this seems to be an issue with the users in the OWASP Zap docker setup, but I didn't seek a workaround very hard.